fetchit_challenge/models/rarelab_long_table/meshes/table_EM.dae

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0" encoding="utf-8"?>
+<COLLADA xmlns="http://www.collada.org/2005/11/COLLADASchema" version="1.4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <asset>
+    <contributor>
+      <author>Blender User</author>
+      <authoring_tool>Blender 2.79.0 commit date:2018-03-22, commit time:14:10, hash:f4dc9f9d68b</authoring_tool>
+    </contributor>
+    <created>2019-04-12T11:28:54</created>
+    <modified>2019-04-12T11:28:54</modified>
+    <unit name="meter" meter="1"/>
+    <up_axis>Z_UP</up_axis>
+  </asset>
+  <library_images>
+    <image id="EM_png" name="EM_png">
```

Comment:
Do a diff with [simple.dae](https://github.com/TheRARELab/fetch_gazebo/blob/gazebo9/fetchit_challenge/models/fetchit_table/meshes/simple.dae) to remove the use of EM logo